editor/src/components/inspector/sections/component-section/property-control-controls.tsx

```diff
@@ -348,8 +331,32 @@ export const NumberInputPropertyControl = React.memo(
       propMetadata.onUnsetValues,
     )
 
+    const value = useEditorState(
+      Substores.metadata,
+      (store) => {
+        // if the target prop is children, dig for its value
+        if (propName === 'children') {
+          const element = MetadataUtils.findElementByElementPath(
+            store.editor.jsxMetadata,
+            props.elementPath,
+          )
+          if (element != null && isRight(element.element) && isJSXElement(element.element.value)) {
+            const child = element.element.value.children[0]
+            if (child.type === 'ATTRIBUTE_OTHER_JAVASCRIPT') {
+              const parsed = parseNumber(child.originalJavascript)
```

Comment:
this kinda sucks a bit, if you have better ideas please do tell!

Reply:
I _think_ the info we are looking for is in the `editorState.allElementProps`! IIRC we store props.children in the spy, it's not filtered out

Reply:
maybe we could even put this into `propMetadata.value` so here you don't have to special case it